test/functional/apps/console/_misc_console_behavior.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const log = getService('log');
+  const browser = getService('browser');
+  const PageObjects = getPageObjects(['common', 'console', 'header']);
+
+  describe('misc console behavior', function testMiscConsoleBehavior() {
+    this.tags('includeFirefox');
+    before(async () => {
+      log.debug('navigateTo console');
```

Comment:
is this needed? 🤔

Reply:
It's not needed since we are logging it in the main entry point. It was for debugging when running the tests in headless mode. I removed all unnecessary log statements from this test file. Thanks for pointing out